templates/statistics/index.html.twig

```diff
@@ -0,0 +1,18 @@
+{# statistics/index.html.twig #}
+
+{% extends 'base.html.twig' %}
+
+{% block page %}
+
+  <h3>Statistiques</h3>
+  <ul>
+    <li><a href='index.php?page=statistiques/temps.php'>Feuille de temps</a></li>
+    <li><a href='index.php?page=statistiques/agents.php'>Par agent</a></li>
+    <li><a href='index.php?page=statistiques/service.php'>Par service</a></li>
+    <li><a href='index.php?page=statistiques/statut.php'>Par statut</a></li>
+    <li><a href='index.php?page=statistiques/postes.php'>Par poste</a></li>
+    <li><a href='index.php?page=statistiques/postes_synthese.php'>Par poste (Synthèse)</a></li>
+    <li><a href='index.php?page=statistiques/postes_renfort.php'>Postes de renfort</a></li>
```

Comment:
Please use asset in href

Reply:
Should I use asset in href despite the absence of some pages for now ?

Reply:
Several pages are symfonized in PR after this current one. Changing links will lead to errors while the new pages aren't merged in master....